packages/e2e-tests/specs/editor/blocks/cover.test.js

```diff
@@ -126,24 +126,20 @@ describe( 'Cover', () => {
 			'.components-circular-option-picker__option-wrapper:first-child button'
 		);
 
-		// Select the cover block.By default the child paragraph gets selected.
+		// Select the cover block. By default the child paragraph gets selected.
 		await page.click(
 			'.edit-post-header-toolbar__document-overview-toggle'
 		);
 		await page.click(
 			'.block-editor-list-view-block__contents-container a'
 		);
 
-		const heightInput = (
-			await page.$x(
-				'//div[./label[contains(text(),"Minimum height of cover")]]/following-sibling::div//input'
-			)
-		 )[ 0 ];
+		const heightInput = await page.waitForSelector(
+			'input[id*="block-cover-height-input"]'
+		);
 
 		// Verify the height of the cover is not defined.
-		expect(
-			await page.evaluate( ( { value } ) => value, heightInput )
-		).toBeFalsy();
+		expect( heightInput.value ).toBeFalsy();
```

Comment:
We could check more precisely for the exact value which is `.toBe( '' )`.

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/47294/commits/9aaa140c87038f5e1d1b3774b05f8ad58fdbb517